tasty-inspector/src/scala/tasty/inspector/TastyInspector.scala

```diff
@@ -62,7 +61,7 @@ trait TastyInspector:
 
     end TastyInspectorPhase
 
-    val currentClasspath = ClasspathFromClassloader(getClass.getClassLoader)
+    val currentClasspath = System.getProperty("java.class.path")
```

Comment:
This class path will not be complete we need to recover the URLs from the URLClassLoders firts